.github/workflows/mediasoup-node.yaml

```diff
@@ -11,12 +11,22 @@ concurrency:
 jobs:
   ci:
     strategy:
+      fail-fast: false
       matrix:
         ci:
           - os: ubuntu-20.04
             node: 18
           - os: ubuntu-22.04
             node: 20
+          - os: ubuntu-24.04
+            node: 20
+            meson_args: '-Db_sanitize=address'
+          - os: ubuntu-24.04
+            node: 20
+            meson_args: '-Db_sanitize=undefined'
```

Comment:
Is `-Db_sanitize=undefined` the same as not passing `-Db_sanitize` argument?

Reply:
No, this is an undefined [behavior sanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html)